src/app/cli/src/tests/full_test.ml

```diff
@@ -50,6 +50,9 @@ let medium_curves = false
 [%%if
 time_offsets = true]
 
+[%%inject
```

Comment:
I think this needs to be hoisted out of the `[%%if ]` above, otherwise it's undefined when `time_offsets = false`

Reply:
Yes, just fixed it. Thank you :)